_api-reference/ingest-apis/processors-reference/append.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: Append
+parent: Ingest processors 
+grand_parent: Ingest APIs
+nav_order: 10
+---
+
+# Append
+
+The `append` processor is used to add values to a field:
+- If the field is an array, the `append` processor appends the specified values to that array.
+- If the field is a scalar field, the `append` processor converts it to an array and appends the specified values to that array.
+- If the field does not exist, the `append` processor creates an array with the specified values.
+
+The syntax for the `append` processor is: 
+
+```json
+{
+    "append": {
+        "field": "field_name",
+        "value": ["value1", "value2", "{{value3}}"]
```

Comment:
Would be better to use something else than `{{value3}}` because it is not the actual value but field name.

Reply:
Revised